enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx

```diff
@@ -39,9 +39,6 @@ const ContentView = ({
     .with("filter", () => (
       <InteractiveQuestion.Filter onClose={onReturnToVisualization} />
     ))
-    .with("summarize", () => (
-      <InteractiveQuestion.Summarize onClose={onReturnToVisualization} />
-    ))
```

Comment:
I checked the docs and we don't mention `onClose` there, so no docs update needed.

Reply:
I am thinking now - maybe I need to decouple the `SummarizePicker` from the `SummarizeDropdown` and keep some of this functionality if we're going to have the composable components.